src/main/kotlin/no/nav/familie/ef/sak/barn/BarnService.kt

```diff
@@ -148,6 +153,14 @@ class BarnService(
         )
     }
 
+    private fun barnFraForrigeBehandling(fagsakId: UUID): List<BehandlingBarn> {
+        val forrigeBehandling = behandlingService.finnSisteIverksatteBehandlingMedEventuellAvslått(fagsakId)
+        feilHvis(forrigeBehandling == null) {
+            "Kan ikke behandle ettersending når det ikke finnes en tidligere behandling"
```

Comment:
Finurlig feilmelding inni en `BarnService` ? Tenker jo at denne feilmeldingen skulle kommet mye tidligere isåfall? Og her burde vi si `Det finnes ingen iverksatte behandlinger` eller noe i den duren?

Reply:
Har endret feilmeldingen og inlineat koden. 
Det finnes egentlige allerdede i JournaøføringRequest:
```kotlin
feilHvis(
            ustrukturertDokumentasjonType == UstrukturertDokumentasjonType.ETTERSENDING &&
                behandling.behandlingstype != BehandlingType.REVURDERING
        ) {
            "Må journalføre ettersending på ny behandling som revurdering"
        }
        feilHvis(
            ustrukturertDokumentasjonType == UstrukturertDokumentasjonType.ETTERSENDING &&
                vilkårsbehandleNyeBarn == VilkårsbehandleNyeBarn.IKKE_VALGT
        ) {
            "Man må velge om man skal vilkårsbehandle nye barn på ny behandling av type ettersending"
        }
```